tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/MetricName.java

```diff
@@ -65,7 +44,7 @@
 
```

Comment:
Actually should make this:

```
    @Value.NaturalOrder
    NavigableMap<String, String> safeTags();
```

Reply:
That would do the trick, if we're okay with exposing `NavigatableMap` instead of `Map`

Reply:
Why does MetricName need to be comparable? Are we better off removing that piece?

Reply:
I mainly added it for convenience for the `toString` form when analyzing `getMetrics` output